docs/concepts/multi-tenancy.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "Multi-tenancy"
+---
+
+TopK does not have a built-in concept of namespaces or tenants. Multi-tenancy is achieved through encoding tenant IDs in documents **`_id`** value.
+
+This design enables TopK to scale a single collection efficiently without performance degradation.
+
+### Storing documents for a specific tenant
+
+To store documents for a specific tenant, prepend the **tenant ID** to the document **`_id`** value:
+
+<CodeGroup>
+
+```python Python
+client.collection("books").upsert([
+    {"_id": "tenant-1234_1", "title": "The Great Gatsby", "author": "F. Scott Fitzgerald"},
+    {"_id": "tenant-1234_2", "title": "To Kill a Mockingbird", "author": "Harper Lee"},
+])
+```
+
+```typescript Javascript
+await client.collection("books").upsert([
+  { _id: "tenant-1234_1", title: "The Great Gatsby", author: "F. Scott Fitzgerald" },
+  { _id: "tenant-1234_2", title: "To Kill a Mockingbird", author: "Harper Lee" },
+]);
+```
+
+</CodeGroup>
+
+### Querying documents for a specific tenant
```

Comment:
we should explain more how this is different from other solutions and how performance improves with this ferature